installer/roles/kubernetes/tasks/main.yml

```diff
@@ -78,20 +78,18 @@
     - postgres_svc_details is defined and postgres_svc_details.rc != 0
     - openshift_host is defined
 
+- name: Render Postgres Values (Kubernetes)
+  set_fact:
+    postgres_values: "{{ lookup('template', 'postgresql-helm-values.yml.j2')"
```

Comment:
Hi and thanks for the PR. I believe we may need to wrap this conditional to ensure that it only runs on vanilla (non-openshift) Kubernetes. The below invocation of `shell` does this, so one option would be to move that condition to a `block` and wrap this new code with the proceeding task.

Reply:
Good catch! Change submitted along with a rendering bugfix in latest commit.